controllers/watcher_common.go

```diff
@@ -47,6 +49,18 @@ const (
 	// WatcherAPILabelPrefix - a unique, service binary specific prefix for the
 	// labels the WatcherAPI controller uses on children objects
 	WatcherAPILabelPrefix = "watcher-api"
+
+	// PrometheusHostKey is the name of the key for the Prometheus hostname in the secret
+	PrometheusHostKey = "prometheus_host"
+
+	// PrometheusPortKey is the name of the key for the Prometheus port in the secret
+	PrometheusPortKey = "prometheus_port"
+
+	// PrometheusTLSKey is the name of the key for the variable defining if TLS is enabled in Prometheus
+	PrometheusTLSKey = "prometheus_tls"
+
+	// PrometheusCaCertKey is the name of the key for the variable defining if TLS is enabled in Prometheus
+	PrometheusCaCertKey = "prometheus_cacert"
```

Comment:
there is now some overlap between this one and https://github.com/openstack-k8s-operators/watcher-operator/pull/40/files#diff-14d76f1f94541be883da14f46430d3de58c97ec4b344863414b5f69fd4c58478 
are they rebased on each other or something

Reply:
Cleaned in my last version. Prometheus stuff will come later.